src/modules/commander/Commander.cpp

```diff
@@ -526,8 +526,6 @@ Commander::Commander() :
 	// default for vtol is rotary wing
 	_vtol_status.vtol_in_rw_mode = true;
 
-	/* init mission state, do it here to allow navigator to use stored mission even if mavlink failed to start */
-	mission_init();
```

Comment:
@dagar @MaEtUgR hope I'm not missing anything, but to me this makes no sense. Mission itself just reads from dataman, right?

Reply:
It makes sense for air in restore to not have to depend on mavlink, but I think at this point we should get rid of it.